release_images.yml

```diff
@@ -41,4 +41,25 @@ release_images:
       cuda_version: "cu113"
       example: False
       disable_sm_tag: False  # [Default: False] This option is not used by Example images
+      force_release: False
+  4:
+    framework: "pytorch"
+    version: "1.13.0"
+    arch_type: "x86"
+    customer_type: "ec2"
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: [ "py39" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu117"
+      example: True
```

Comment:
Example images are only gpu, and require a separate image definition.

You would need to do something like this to enable example image while also preserving the existing EC2 training cpu/gpu images

```
  4:
    framework: "pytorch"
    version: "1.13.0"
    arch_type: "x86"
    customer_type: "ec2"
    training:
      device_types: ["cpu", "gpu"]
      python_versions: [ "py39" ]
      os_version: "ubuntu20.04"
      cuda_version: "cu117"
      example: False
      disable_sm_tag: False  # [Default: False] This option is not used by Example images
      force_release: False
    inference:
      device_types: ["cpu", "gpu"]
      python_versions: ["py39"]
      os_version: "ubuntu20.04"
      cuda_version: "cu117"
      example: False
      disable_sm_tag: False  # [Default: False] This option is not used by Example images
      force_release: False
  5:
    framework: "pytorch"
    version: "1.13.0"
    arch_type: "x86"
    customer_type: "ec2"
    training:
      device_types: ["gpu"]
      python_versions: [ "py39" ]
      os_version: "ubuntu20.04"
      cuda_version: "cu117"
      example: True
      disable_sm_tag: False  # [Default: False] This option is not used by Example images
      force_release: False
```

Reply:
TY.. updating